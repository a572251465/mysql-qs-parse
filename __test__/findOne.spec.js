import MysqlParse from '../src'

describe('findOne 测试', () => {
  let db = null
  test('findOne 测试数据', async () => {
    db = new MysqlParse({
      host: 'localhost',
      user: 'root',
      password: 'location@root',
      database: 'super-admin-system'
    })

    await db.open()
    const res = await db.findOne(['type', 'visitNum'], 'cuVisitNum', {
      type: 'work-platform'
    })

    // 利用快照的形式 对数据进行模拟测试
    expect(res).toMatchSnapshot({
      type: 'work-platform',
      visitNum: expect.any(Number)
    })
  })

  afterAll(() => {
    db && db.release()
  })
})
