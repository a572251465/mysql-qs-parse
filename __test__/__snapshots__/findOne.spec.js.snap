// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`findOne 测试 findOne 测试数据 1`] = `
Object {
  "type": "work-platform",
  "visitNum": Any<Number>,
}
`;
